src/astx/operators.py

```diff
@@ -128,3 +131,67 @@ def get_struct(self, simplified: bool = False) -> ReprStruct:
         key = str(self)
         value = self.value.get_struct(simplified)
         return self._prepare_struct(key, value, simplified)
+
+
+@public
+@typechecked
+class AugAssign(DataType):
+    """AST class for augmented assignment (+= , -= , |=, *= , /= , //= , eg)."""
+
+    OPERATORS = {
```

Comment:
probably this is not necessary, because you are already using Literal in the type annotation, and typechecked will check that for you for free